### ISSUE TITLE
2.8.3 Patch Release

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3024]
- #2956 
- #2934 
- #2942 
- #2977
- #2969 
- #2995
- #2996
- #3009 
- #3000

## Pick list
```
git cherry-pick cc7784013e4048ab39857b470e2a58d73c454513
git cherry-pick e20b456a98cd5061b2f600bed0fc37b5a1d35d09
git cherry-pick 7ecb796ae99e3f38bd1005f5e8bbef84ab90fa12
git cherry-pick 51cb17d95052d1a9d2cb21cda87508ac46c685bc
git cherry-pick 7bda10491b322f808803a4f94325f59fd22f95a1
git cherry-pick ad21a7957e3ceb9d995adf4ddc8190dfb6428c7c
git cherry-pick f78adde9e6a69ac6d8effac5de8b1b72ed7d272a
git cherry-pick f29714ab87dabca0db1fb7ad69946bee0343b500
git cherry-pick 931431a7ed7ba71068b1b5afc3abb3bfc383b09f
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.8.2..HEAD --pretty='format:- %s'`
```
- Update drupal/core-recommended requirement from 10.1.6 to 10.1.7 (#3000)
- Closes #3008: Update seboettg/citeproc-php requirement from 2.6.0 to 2.6.2 (#3009)
- #2084 Add media_library_theme_reset to list of deprecated Quickstart modules. (#2996)
- Closes #1946 Embedded images in demo content are not found (#2995)
- Closes #2852 Deprecation warnings while news feed is importing (#2969)
- Closes #1675 Minor update to help text for "Where" section of Event Page (#2977)
- Update citation-style-language/styles requirement from 0.0.303 to 0.0.325 (#2942)
- Close #2933 Update external link field help text to mention University calendar, and to be more concise. (#2934)
- Closes #2169 Close icon broken in media library edit module's modal (#2956)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.8.3
